client/my-sites/plans/current-plan/index.jsx

```diff
@@ -125,7 +124,7 @@ class CurrentPlan extends Component {
 					/>
 				) }
 
-				<ProductPurchaseFeatures>
+				<div className="current-plan__product-purchase-features">
```

Comment:
If we're not using this `className` now (for styling or otherwise), let's actually remove it (because YAGNI).
I guess we could maybe even remove the `<div />` wrapping and replace it by `<Fragment />` instead(?)

Reply:
nice catch 👍